cedar-testing/src/cedar_test_impl.rs

```diff
@@ -300,6 +304,34 @@ impl CedarTestImplementation for RustEngine {
             response,
             timing_info: HashMap::from([("authorize".into(), Micros(duration.as_micros()))]),
         };
+
+        // now check that we get the same response with entity manifest, as long as the schema is valid
+        let validator = Validator::new(schema.clone());
+        let validation_result = validator.validate(policies, ValidationMode::Strict);
+        if validation_result.validation_passed() {
+            let entity_manifest_res = compute_entity_manifest(schema, policies);
+
+            match entity_manifest_res {
+                Ok(entity_manifest) => {
+                    // PANIC SAFETY: slicing should succeed for tests
+                    #[allow(clippy::expect_used)]
+                    let entity_slice = entity_manifest
```

Comment:
Are we allowed to panic in the testing infra here? We could return a new kind of TestResult or something like that.
Or perhaps I'm abusing the `is_authorized` function

Reply:
Edit: Removed use of panic